.gitmodules

```diff
@@ -1,12 +1,6 @@
 [submodule "Carthage/Checkouts/xcconfigs"]
 	path = Carthage/Checkouts/xcconfigs
 	url = https://github.com/jspahrsummers/xcconfigs.git
-[submodule "Carthage/Checkouts/Nimble"]
-	path = Carthage/Checkouts/Nimble
-	url = https://github.com/Quick/Nimble.git
-[submodule "Carthage/Checkouts/Quick"]
-	path = Carthage/Checkouts/Quick
-	url = https://github.com/Quick/Quick.git
 [submodule "Carthage/Checkouts/Swinject"]
 	path = Carthage/Checkouts/Swinject
 	url = https://github.com/Swinject/Swinject.git
```

Comment:
Might be a good idea to migrate this away from submodules and just use Carthage.

Reply:
I think we've already done this for the other repositories.

Reply:
Created https://github.com/Swinject/Swinject/issues/481

Reply:
Yes, the other repos migrated from submodules to typical Carthage builds. Only this repo still uses the submodules. Thank you for making the ticket😃